### ISSUE TITLE
Cannot catch error in stream. Error: ffmpeg exited with code 1: pipe:0: Invalid data found when processing input

### ISSUE THREAD

**Issue by alisamaiskaya** [fluent-ffmpeg/node-fluent-ffmpeg#1065]
#430 # Version information

* fluent-ffmpeg version: ^2.1.2
* ffmpeg version: -
* OS: Mac OS

### Code to reproduce

```js
// Converter-function
function videoConverter(input) {
  try {
    const pt = ffmpeg(input)
      .addOutputOptions('-movflags +frag_keyframe+separate_moof+omit_tfhd_offset+empty_moov')
      .format('mp4')
      .pipe();

    return { pt };
  } catch (error) {
    Log.error({
      message: 'Cannot convert video',
      error,
    });

    throw error;
  }
}

// Common function for downloading content as stream, convertes it etc.
async function convertVideo(sha1, sizeLimit) {
  try {
    // It's pool for converting
    const data = await withPoints(10, async (converters) => {
      // Function for getting first stream chunk
      const waitForChunks = defer();
      // Download content from store as stream
     // Step 1
      const stream = await getContentAsStream(sha1);
      // Converted our video
      // Step 2
      const { pt } = converters.videoConverter(stream);
      // This class helps to get two first chunks of stream after converting and to start asynchronus uploading converted content to store
      const lengthLimitTransform = new StreamLengthLimit({ sizeLimit });
      const newStream = pt
        .pipe(new BufferFirst())
        .pipe(lengthLimitTransform);

      // Resolve promise after getting the second chunk
      newStream.once('data', waitForChunks.resolve);

      // I see no errors in terminal in these streams, but I have error in terminal in some stream!!
      stream.on('error', (e) => {
        console.log('ERROR1', e);
        waitForChunks.reject(e);
      });
      newStream.on('error', (e) => {
        console.log('ERROR2', e);
        waitForChunks.reject(e);
      });
      pt.on('error', (e) => {
        console.log('ERROR3', e);
        waitForChunks.reject(e);
      });

      // It's timeout if we have no second chunck
      const timer = setTimeout(() => {
        waitForChunks.reject(new Error('Timeout is up'));
      }, constants.secondChunkTimeout);

      await waitForChunks.promise;
      clearTimeout(timer);

      // Upload to store
      const convertedSha = await uploadContent(newStream);
      const contentSize = lengthLimitTransform.getLength();

      return {
        convertedSha,
        contentSize,
      };
    });

    return {
      ...data,
      status: 'converted',
      contentType: 'video/mp4',
    };
  } catch (error) {
    Log.error({
      message: 'Cannot convert video',
      function: 'convertVideo',
      sha1,
      error,
    });

    return {
      convertedSha: null,
      status: 'failed',
      contentSize: 0,
      contentType: '',
    };
  }
}
```

(note: if the problem only happens with some inputs, include a link to such an input file)

### Expected results
I want to convert video with 
file extension ".mov" ( I have no the same problems with another video files)
I download the file from store as stream (Step 1). 

### Observed results
But after Step 2 I have next result in terminal:

throw er; // Unhandled 'error' event
      ^

Error: ffmpeg exited with code 1: pipe:0: Invalid data found when processing input
Cannot determine format of input stream 0:0 after EOF
Error marking filters as finished
Conversion failed!

I understand there is error in some stream. But .on "error" couldn't help me to catch this error.

**Comment by FedericoCarboni** [comment]
@alisamaiskaya Duplicate of https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/issues/932#issuecomment-699675713
